home/templates/wagtailsnippets/snippets/index.html

```diff
@@ -21,7 +21,7 @@
             {% block import_button %}
                 <div class="actionbutton">
                         <div class="dropdown dropdown-button match-width col">
-                            <a href="{% url 'import' %}" class="button bicolor button--icon">{% icon name="download" wrapped=1 %}{% trans 'Import' %}</a>
+                            <a href="{% url 'import_assessment' %}" class="button bicolor button--icon">{% icon name="download" wrapped=1 %}{% trans 'Import' %}</a>
```

Comment:
Won't this change it for all snippets? ie. it will also change it for ordered content sets, not just for assessments